SwiftAppTemplate/SwiftAppTemplateTests/SwiftAppTemplateTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SwiftAppTemplateTests.swift
```

Comment:
For simplicity / clarity, maybe we should delete the test target entirely (and this file?)

Reply:
Done!